src/julia/libjulia.py

```diff
@@ -210,21 +211,27 @@ def __init__(self, libjulia_path, bindir, sysimage):
         if sys.version_info >= (2, 7, 13) and sys.version_info < (2, 7, 14):
             libjulia_path = libjulia_path.encode("ascii")
 
+        with self._windows_pathhack():
+            self.libjulia = ctypes.PyDLL(libjulia_path, ctypes.RTLD_GLOBAL)
+
+        setup_libjulia(self.libjulia)
+
+    @contextmanager
+    def _windows_pathhack(self):
```

Comment:
maybe you can use `if os.name == 'nt'` here.

Reply:
Good point! Addressed in f05c3a6dc7d4a5973fb05820fb9d21946929b266